app.js

```diff
@@ -1,46 +1,110 @@
 let fs = require('fs'),
     mysql = require('mysql'),
-    parse = require('csv-parse');
+    parse = require('csv-parse'),
+    filesToInsert,
+    isUserSpecified = false;
+
 
 const DB_CONFIG = require('./dbconfig.json');
+var connection = mysql.createConnection(DB_CONFIG);
```

Comment:
This connection doesn't look like it's being closed if there are no files. It also looks like it will be closed for the first file that finishes processing (and will therefore cause errors when you are importing multiple files).